src/__tests__/migrations/1586494851710-seed_data_test.ts

```diff
@@ -0,0 +1,135 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+import { CategoryEntity } from "../category.entity";
+import { CommentEntity } from "../comment.entity";
+import { PostEntity } from "../post.entity";
+
+let seedDatas = [
```

Comment:
Can we make these dynamic within the tests? So we can test different amounts etc?

Reply:
With a clean up for after all tests as well please :)

Reply:
Yup. Have added **postAmount**. It can be dynamic when run test cases.  Also, cleanUp data after tests done.